application-development/helloworld-springboot-auth/src/main/java/com/example/java/gettingstarted/WebSecurityConfig.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.java.gettingstarted;
+
+import org.springframework.boot.autoconfigure.security.oauth2.client.EnableOAuth2Sso;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@EnableOAuth2Sso
+@Configuration
+public class WebSecurityConfig extends WebSecurityConfigurerAdapter {
```

Comment:
This is based off https://github.com/GoogleCloudPlatform/getting-started-java/tree/master/helloworld-springboot , if one wants to  try out the unprotected end points.